### ISSUE TITLE
PostProcessResources cannot be built into the scene asset bundle

### ISSUE THREAD

**Issue by xophiix** [Unity-Technologies/PostProcessing#682]
##### What happened?
PostProcessResources referenced in the scene cannot be built into the scene asset bundle.
##### Post-processing stack version (v1, v2)?
v2
##### Unity version, operating system, target platform (standalone windows, mac, iOS, PS4...)?
2018.2.0f2, Windows,  Standalone windows.

Please suggest me a work around for this, it's urgent

**Comment by BernsteinA** [comment]
```
using UnityEngine;
using UnityEngine.Rendering.PostProcessing;

// Put this in your first scene so that the post-processing shaders get added into a build
public class PostProcessResourcesReferencer : MonoBehaviour {
    [SerializeField] PostProcessResources postProcessResources;
}
```
cc #623 #668 #616 #568 and possibly #553

**Comment by naashw** [comment]
hi,
i just put this on the first scene ?  and let **postProcessResources** empty ?

**Comment by BernsteinA** [comment]
No, you'll need to hook it up in the inspector to point to the PostProcessResources asset in your project.
